README.adoc

```diff
@@ -232,19 +232,19 @@ Event.java
 include::finish/backendServices/src/main/java/io/openliberty/guides/event/models/Event.java[tags=**;!copyright]
 ----
 
-* The [hotspot=25-27 file=0]`updateEvent()` method creates a managed instance of a detached entity instance. 
+* The [hotspot=updateEvent file=0]`updateEvent()` method creates a managed instance of a detached entity instance. 
 The entity manager automatically tracks all managed entity objects in its persistence context 
 for changes and synchronizes them with the database. However, if an entity becomes detached, 
-you must merge that entity into the persistence context by calling the [hotspot=26 file=0]`merge()` method 
+you must merge that entity into the persistence context by calling the [hotspot=Merge file=0]`merge()` method 
 so that changes to loaded fields of the detached entity are tracked.
 
-* The [hotspot=29-31 file=0]`deleteEvent()` method removes an instance of the [hotspot=17-70 file=1]`Event` entity class from the database by 
-calling the [hotspot=30 file=0]`remove()` method on an `EntityManager` instance. The state of the entity is
+* The [hotspot=deleteEvent file=0]`deleteEvent()` method removes an instance of the [hotspot=Event file=1]`Event` entity class from the database by 
+calling the [hotspot=Remove file=0]`remove()` method on an `EntityManager` instance. The state of the entity is
 changed to removed and is removed from the database upon transaction commit. 
 
-The DAO is injected into the [hotspot=28-128 file=2]`backendServices/src/main/java/io/openliberty/guides/event/resources/EventResource.java`
-class and used to access and persist data. The [hotspot=39 file=2]`@Transactional` annotation is used in the
-[hotspot=28-128]`EventResource` class to declaratively control the transaction boundaries on the [hotspot=26 file=2]`@RequestScoped` CDI bean.
+The DAO is injected into the [hotspot=DAO file=2]`backendServices/src/main/java/io/openliberty/guides/event/resources/EventResource.java`
+class and used to access and persist data. The [hotspot=Transactional file=2]`@Transactional` annotation is used in the
+[hotspot=EventResource]`EventResource` class to declaratively control the transaction boundaries on the [hotspot=RequestedScoped file=2]`@RequestScoped` CDI bean.
```

Comment:
`EventResource` hotspot is wrong
![image](https://user-images.githubusercontent.com/26514093/66072634-673a4980-e523-11e9-87b3-d116e3f262d1.png)